examples/chat_history.py

```diff
@@ -1,49 +1,106 @@
 """A basic example showcasing how to manually manage chat history."""
 import os
 
```

Comment:
this has become complicated for no value add. Please revert to previous example but using OpenAIPrompt.

Reply:
Resolved.